oci-test/tests/oci_logging_log/variables.tf

```diff
@@ -29,13 +34,26 @@ provider "oci" {
 
 resource "oci_logging_log_group" "named_test_resource" {
   compartment_id = var.tenancy_ocid
-  display_name = var.resource_name
+  display_name   = var.resource_name
+}
+
+locals {
+  path = "${path.cwd}/output.json"
 }
 
-resource "oci_logging_log" "named_test_resource" {
-  log_group_id = oci_logging_log_group.named_test_resource.id
-  display_name = var.resource_name
-  log_type = "Custom log"
+resource "null_resource" "named_test_resource" {
+  depends_on = [oci_logging_log_group.named_test_resource]
+  provisioner "local-exec" {
+    command = "oci logging log create --display-name ${var.resource_name} --log-group-id ${oci_logging_log_group.named_test_resource.id} --log-type ${var.log_type}"
```

Comment:
Is resource getting deleted as we are creating resource from CLI ?

Reply:
updated with posttest tf.